pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -249,6 +340,38 @@ type AttachedVolume struct {
 	DevicePath string
 }
 
+// GenerateMsgDetailed returns detailed msgs for attached volumes
+func (volume *AttachedVolume) GenerateMsgDetailed(prefixMsg, suffixMsg string) (detailedMsg string) {
+	detailedStr := fmt.Sprintf("(UniqueName: %q) on node %q", volume.VolumeName, volume.NodeName)
+	volumeVolumeSpecName := "nil"
```

Comment:
I would probably just call it volumeSpecName.

This nil check should be added for the other interface implementations too.